### PR TITLE
Game INI update for Tron Evolution Battle Grids (all editions)

### PR DESCRIPTION
**Pull request by gordonfreeman01**
Automatically set Texture Cache Accuracy to safe in game INI as per wiki entry:
https://wiki.dolphin-emu.org/index.php?title=Tron:_Evolution_-_Battle_Grids

### CODE REVIEW

File: Data/Sys/GameSettings/STR.ini [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# STRE4Q - Tron: Evolution - Battle Grids (NTSC-U)
+# STRP4Q - Tron: Evolution - Battle Grids (PAL)
+# STRX4Q - Tron: Evolution - Battle Grids Championship Edition (NTSC-U)
```
Comment (JMC47) [inline Data/Sys/GameSettings/STR.ini:3]: Please change this to

# STRE4Q, STRP4Q, STRX4Q - Tron Evolution - Battle Grids

This is more in-line with how we format GameINIs, I'm not on my computer, but you can check other 3 letter INIs to see how it is normally handled.  I know that one of these are a slightly different name, I think there's a Shawn White game or something with a similar edition, not sure how we handle that.

**Review by JMC47 — COMMENTED** [review]
Other than the formatting for the name, looks fine to me.

**Comment by JMC47** [comment]
One last thing is that there's no new-line at the end of the file.  Not sure how important that is, but the review thing complained.

File: Data/Sys/GameSettings/STR.ini (hunk 1 above)
Comment (gordonfreeman01) [reply Data/Sys/GameSettings/STR.ini]: Fixed, "Shaun White Snowboarding: Road Trip" has a different code (the last letter is different) between the normal and Target Limited Edition so it has a different entry for both entries. For this game I've combined everything as you suggested.

Comment (JMC47) [reply Data/Sys/GameSettings/STR.ini]: Sounds fine.  If we add 6 letter INI, we can put the full name for Championship Edition when it comes.

**Comment by gordonfreeman01** [comment]
Cool, added the newline

**Comment by JosJuice** [comment]
Please squash the two commits into one. Looks good to merge after that.

**Comment by JMC47** [comment]
You need to squash the changes into one commit, then I can merge.

**Comment by gordonfreeman01** [comment]
Squashed, sorry, forgot to do that after adding the newline.

**Comment by JMC47** [comment]
@dolphin-emu-bot rebuild

**Pull request merged**